docs/webpack.dev.config.js

```diff
@@ -25,43 +28,70 @@ const config = decorateClientConfig({
   },
 
   module: {
-    loaders: [
+    rules: [
       {
         test: /\.js$/,
-        loader: 'babel',
-        query: babelConfig,
-        include: appPaths
+        include: appPaths,
+        use: {
+          loader: 'babel-loader',
+          options: babelConfig
+        }
       },
       {
         test: /\.js$/,
-        loader: 'babel',
-        query: {
-          babelrc: false,
-          presets: ['es2015']
-        },
-        include: /node_modules/
+        include: /node_modules/,
+        use: {
+          loader: 'babel-loader',
+          options: {
+            babelrc: false,
+            presets: ['es2015']
+          }
+        }
       },
       {
         test: /\.less$/,
-        loader: 'style!css?modules&localIdentName=[name]__[local]___[hash:base64:5]!postcss!less',
-        include: appPaths
+        include: appPaths,
+        use: [
+          {
+            loader: 'style-loader'
+          },
+          {
+            loader: 'css-loader',
+            options: {
+              modules: true,
+              localIdentName: '[name]__[local]___[hash:base64:5]'
+            }
+          },
+          {
+            loader: 'postcss-loader',
+            options: {
+              plugins: [autoprefixer]
```

Comment:
Is there a reason for not passing the [`autoprefixer.config.js`](config/autoprefixer.config.js) as you do [here](https://github.com/seek-oss/seek-style-guide/pull/341/files#diff-242962979875c719b0652867bfbe48f2R66)

Reply:
This was a straight port, not sure what the original reasoning was, if there was any. Makes sense to pass it in here too 👍

Reply:
Also further to this we are now [exporting the autoprefixer config from the seek-style-guide-webpack module](https://github.com/seek-oss/seek-style-guide-webpack/blob/master/autoprefixer.config.js), so might be a good clean up to move onto that one given they are identical?

Reply:
As discussed offline, let's come back to this clean up, as the autoprefixer config should really be its own module.